server/src/core/util/validation-util.ts

```diff
@@ -32,12 +33,11 @@ export const phoneValidationSchema = joi.string()
 
 export const passwordValidationSchema = joi.string()
   .required()
-  .pattern(/^.{8,18}$/)
   .messages({
     'any.required': 'Password is required',
     'string.base': 'Invalid type, password must be a string',
     'string.empty': 'Please enter your password',
-    'string.pattern.base': 'Invalid password. Must range between 8 and 18 characters'
+    'string.pattern.base': 'Invalid password. Must be at least one character'
```

Comment:
Is this for login or sign up? I don't think we're supposed to be changing password validation when signing up. And if it's for logging in, I don't think we're supposed to validate the password beyond the fact that it should be a non-empty string.